### ISSUE TITLE
Sub-patterns of PropertyPattern should be on same line when inside conditions

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1494]
We should issue a warning in case the sub-patterns of a property pattern span multiple lines.

Following code should trigger a warning:
```c#
while (xmlReader.Read())
{
    if (xmlReader is {
                          NodeType: XmlNodeType.Element,
                          Name: "Log"
                     })
    {
        return true;
    }
}
```
It should be fixed into:
```c#
while (xmlReader.Read())
{
    if (xmlReader is { NodeType: XmlNodeType.Element, Name: "Log" })
    {
        return true;
    }
}
```
